models/index.js

```diff
@@ -7,6 +7,9 @@ const basename = path.basename(__filename);
 const env = process.env.NODE_ENV || 'development';
 const config = require(__dirname + '/../config/config.json')[env];
 const db = {};
+const twilio = require('../lib/twilio');
```

Comment:
Sequelize autogenerated this file. I think we should leave it and not touch it besides the dirname fix. I generally avoid touching autogenerated files/config because I want it to be transparent which code is mine and which code the machine wrote for me.

Reply:
Where would you prefer this go?

Reply:
I would prefer the twilio module abstracts it away and we don't need a setup function :) I think we can instantiate it in the twilio.js file or use the `getClient()` function to get the client or create a new one.